contrib/debian/copyright

```diff
@@ -1398,51 +1394,3 @@ License: Apache-Qpid-Proton-with-BSD-Subcomponents
    See the License for the specific language governing permissions and
    limitations under the License.
  .
```

Comment:
You can delete the whole of `License: Apache-Qpid-Proton-with-BSD-Subcomponents` (lines 1194..1413).